.github/ISSUE_TEMPLATE.md

```diff
@@ -1,9 +1,13 @@
 <!--
 Thank you for reporting an issue.
 
+*IMPORTANT* - please search our issue tracker *first* to ensure the
```

Comment:
We already have `Please review the guidelines for contributing to this repository.` message on top of new issue page. @dinatale2 agree with you.